setup.cfg

```diff
@@ -93,7 +93,7 @@ extras = True
 # in order to write a coverage file that can be read by Jenkins.
 # TODO (gmuraru) change cov-fail-under as soon as possible
 addopts =
-    --cov syft --cov-report term-missing --cov-fail-under 82 --no-cov-on-fail
```

Comment:
womp womp

Reply:
Sorry, I didn't see that revert in the rebase... 😬